### DISCUSSION TITLE
Can we use Fluent UI react with C++ application?

### DISCUSSION THREAD

**Discussion by harshith187** [microsoft/fluentui#28149] [Q&A]
Hi,

I was reading about how Office apps are built using and got to know that it is built using Fluent UI react V8. I am working on a C++ application, Can fluent UI react be used for UI in C++ application?

And I also went through `Excel.exe.mannifest` file in my system  and noticed that Excel is win32 style application.
![image](https://github.com/microsoft/fluentui/assets/29576770/2f21d51c-1a24-415c-8a57-bdf457e19f94)

So is wanted to understand how to create a C++ Win32 style app and use Fluent UI React in it.

Thank you.

**Comment by AtishayMsft** [comment]
You might want to explore fluentui-react-native for native controls. https://github.com/microsoft/fluentui-react-native
